.github/workflows/add-dependabot-pr-to-project.yml

```diff
@@ -0,0 +1,17 @@
+name: Add Dependabot PR's to project
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  add-to-project:
+    name: Add dependabot pr to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/AxonFramework/projects/2
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Did you add a new secret? Otherwise, I think we can use the default `secrets.GITHUB_TOKEN` token we use for the other GHA's.

Reply:
let me chip in as I did this for Cloud... this token needs `write` access
you can easily copy the token we used on Cloud that was created for this purpose =)

so:
- copy the token and update the file (anyone can do it)
- share the org secret with this repo (only org admins can do it)

Reply:
Cloud is under the AxonIQ organization, not the Framework organization. Hence, sharing is not an option.